rds.tf

```diff
@@ -1,12 +1,48 @@
-resource "aws_db_instance" "default" {
-  allocated_storage     = 10
-  max_allocated_storage = 100
-  engine                = "mysql"
-  engine_version        = "5.7"
-  instance_class        = "db.t3.micro"
-  name                  = "mydb"
-  username              = "foo"
-  password              = "foobarbaz"
-  parameter_group_name  = "default.mysql5.7"
-  skip_final_snapshot   = true
-}
\ No newline at end of file
+resource "aws_db_instance" "mydb1" {
+  allocated_storage        = 256 # gigabytes
+  backup_retention_period  = 7   # in days
+  db_subnet_group_name     = var.rds_public_subnet_group
```

Comment:
vpc, subnets are being created in main.tf, public subnets must be a look-up from main.tf